### PR TITLE
First part settings and header

### PR DESCRIPTION
**Pull request by Olexandr-Sh**
In this part of work, I've customised navigation, header and first part of hero section.
The second and third parts of the project are still not ready.

### CODE REVIEW

File: css/main.css [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+ol {
+  margin: 0;
+  padding: 0;
+  list-style-type: none;
+}
+body {
+  font-family: "Rubik", sans-serif;
+}
+.logo-icon {
+  height: 36.23px;
```
Comment (Plakhotenko) [inline css/main.css]: Could you please round this value.

File: css/main.css [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+  outline: none;
+  cursor: pointer;
+  transition: ease-out 0.3s;
+}
+.sign-up-big-header-btn:hover {
+  background: #f53855;
+  color: #fff;
+  transition: ease-out 0.3s;
+  box-shadow: 0 0 8px #f38f9e;
+  transition: 0.5s all ease;
```
Comment (Plakhotenko) [inline css/main.css]: Please don't set transition for all the properties.

File: css/main.css [hunk 3]
```diff
@@ -0,0 +1,229 @@
...
+}
+.sign-up-big-header-btn:focus {
+  background: #f53855;
+  color: #fff;
+  transition: ease-out 0.3s;
+  box-shadow: 0 0 8px #f38f9e;
+  transition: 0.5s all ease;
```
Comment (Plakhotenko) [inline css/main.css]: The same. Check all the styles.

File: css/main.css [hunk 4]
```diff
@@ -0,0 +1,229 @@
...
+  top: 0;
+  left: 50%;
+  right: 50%;
+  bottom: 0;
+  opacity: 0;
+  content: "";
+}
+.big-btn-hsection:hover::before {
+  transition: 0.5s all ease;
```
Comment (Plakhotenko) [inline css/main.css]: We already have transition for this element. Remove.

File: css/main.css (hunk 1 above)
Comment (Olexandr-Sh) [reply css/main.css]: I've changed to height:36px

**Review by Plakhotenko — APPROVED** [review]

**Pull request merged**